samples/subsys/ipc/ipc_service/static_vrings/remote/src/main.c

```diff
@@ -233,14 +244,32 @@ static void ipc1_entry(void *dummy0, void *dummy1, void *dummy2)
 	k_sem_take(&ipc1_bound_sem, K_FOREVER);
 
 	while (message < 99) {
+		uint32_t len = sizeof(message);
+		void *data;
+
 		k_sem_take(&ipc1_data_sem, K_FOREVER);
-		message = ipc1_received_data;
+
+		message = *((uint8_t *) recv_data);
```

Comment:
Any reason to make a copy? why not print it, copy in TX, and then release the buffer?
As it is a synchronous "request-answer" should not lead to a lock

Reply:
definitely makes sense.